app-modules/authorization/src/Filament/Resources/RoleResource/Pages/ListRoles.php

```diff
@@ -63,6 +69,42 @@ public function getTabs(): array
 
     public function table(Table $table): Table
     {
+        $duplicateRoleAction = fn (): Action => Action::make('duplicateRole')
```

Comment:
What is the purpose of this beign defined in a variable and then used on line `138` like this? It can just be defined inline.